ci-operator/step-registry/openshift/e2e/aws/workers-rhel7/openshift-e2e-aws-workers-rhel7-workflow.yaml

```diff
@@ -0,0 +1,11 @@
+workflow:
+  as: openshift-e2e-aws-workers-rhel7
+  steps:
+    pre:
+    - chain: ipi-aws-pre-workers-rhel7
+    test:
+    - ref: openshift-e2e-test
+    post:
+    - chain: ipi-aws-post
```

Comment:
so all the machines started as part of rhel-scaleup are removed by installer destroy?

Reply:
Yes.